### PR TITLE
chore(k8s-infra): bump to 0.11.11

### PR DESCRIPTION
**Pull request by grandwizard28**
## Summary

- **New Features**
	- Updated the Helm chart version for `k8s-infra` to 0.11.11

### CODE REVIEW

File: charts/k8s-infra/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: k8s-infra
 description: Helm chart for collecting metrics and logs in K8s
 type: application
-version: 0.11.10
+version: 0.11.20
```
Comment (prashant-shahi) [inline charts/k8s-infra/Chart.yaml:5]: shouldn't this be `0.11.11`? :eyes:

Comment (prashant-shahi) [reply charts/k8s-infra/Chart.yaml:5]: There is only single change with kubeletstats metrics preset

Comment (grandwizard28) [reply charts/k8s-infra/Chart.yaml:5]: @coderabbitai please bump the version to 0.11.11 instead of 0.11.20

**Comment by grandwizard28** [comment]
https://github.com/coderabbitai please bump the version to 0.11.11 instead of 0.11.20

**Review by prashant-shahi — APPROVED** [review]

**Pull request merged**